packages/FieldTypes/src/components/Numeric/Numeric.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Container from "../Container";
+import types from "../../types";
+
+// This let mimic the use of Field.types.align.LEFT in documentation and d.ts files
+// not other reason to have it
+const FieldTypes = {};
```

Comment:
@karencumlat @allison-c this is how I was able to display FieldTypes.types.align.LEFT in documentation (reademe and d.ts) I'm not totally happy about it but I think works pretty well for the moment :/

Reply:
Yes, totally! I also left some comments here https://github.com/acl-services/paprika/pull/727#discussion_r496134630